data/trie/interceptedNode.go

```diff
@@ -17,9 +16,9 @@ var _ process.InterceptedData = (*InterceptedTrieNode)(nil)
 
 // InterceptedTrieNode implements intercepted data interface and is used when trie nodes are intercepted
 type InterceptedTrieNode struct {
-	node node
-	SerializedNode
-	hash []byte
+	node           node
+	SerializedNode []byte
```

Comment:
please unexport this, you have getter & setter

Reply:
Done.